src/models/health.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const db = require('../lib/db')
+
+function * getDbHealth () {
+  const isConnected = yield db.isConnected()
+  return isConnected ? {
+    status: 'OK'
+  } : {
+    status: 'KO'
```

Comment:
It depends. If the component waits for something `WAITING` seems to make sense. If there was an unrecoverable error then `NOT OK` does. 
Definition for these status string should maybe be moved to shared so that we enforce the same convention on all components? We can postpone the move to shared though, it doesn't seem high priority.